pom.xml

```diff
@@ -104,7 +104,7 @@
         <org.eclipse.persistence.version>2.5.2</org.eclipse.persistence.version>
         <pdfbox.version>1.8.6</pdfbox.version>
         <quartz.version>1.8.4</quartz.version>
-        <rice.version>2.5.2-r49495</rice.version>
+        <rice.version>2.5.2-20150127-03d707e6da</rice.version>
```

Comment:
should we exclude the oracle specific jar so our oracle profile works correctly?